mmdet3d/datasets/pipelines/transforms_3d.py

```diff
@@ -190,6 +192,144 @@ def __repr__(self):
         return repr_str
 
 
+@PIPELINES.register_module()
+class MultiViewWrapper():
+    """Wrap transformation from single-view into multi-view.
+
+    Args:
+        transform (dict): A dict for specifying the transformation for
+        single-view situation.
+        collected_keys(list[str]): Collect information in transformation
+        like rotate angles, crop roi, and flip state.
+    """
+
+    def __init__(self, transform, collected_keys=[]):
+        _, key = PIPELINES.split_scope_key(transform['type'])
+        library = PIPELINES if key in PIPELINES._module_dict.keys() \
+            else MMDET_PIPELINES
+        self.t = build_from_cfg(transform, library)
+        self.collected_keys = collected_keys
+
+    def __call__(self, input_dict):
+        for key in self.collected_keys:
+            input_dict[key] = []
+        for img_id in range(len(input_dict['img'])):
+            process_dict = dict(img=input_dict['img'][img_id])
+            process_dict = self.t(process_dict)
+            input_dict['img'][img_id] = process_dict['img']
+            for key in self.collected_keys:
+                input_dict[key].append(process_dict[key])
+        return input_dict
+
+
+@PIPELINES.register_module()
+class RandomCrop(MMDET_RandomCrop):
+    """Randomly crop image-view objects under a limitation of range.
+
+    Args:
+        range (tuple[float]): Range of random crop in proportion. (y_min,
+        y_max, x_min, x_max) in [0, 1.0]. Default to (0.0, 1.0, 0.0, 1.0),
+        which is the default setting in RandomCrop. (0.0, 1.0, 0.0, 1.0),
```

Comment:
No need for this sentence?
Just remind "Defaults to (0.0, 1.0, 0.0, 1.0)"
Besides, how about split x and y range to make it clear? and I think relative_x_offset and relative_y_offset would be better?